### PR TITLE
Fix Styling Bugs

### LINKED ISSUE #543: Classifier needs to respond to narrow screen widths
**Package**

lib-classifier

Screenshot 2019-03-13 21 30 39

### LINKED ISSUE #561: Styling bug in task area
**Package**

lib-classifier

**Feature or Issue Description**

The task area height matches that of the subject viewer, instead of matching its own content.

Screenshot 2019-03-18 10 41 58

### PR DESCRIPTION
**Pull request by wgranger**
Package:
lib-classifier

Closes #543 
Closes #561 

Describe your changes:
Fixes a couple bugs in the classifier. The first of which allows the task area to wrap under the subject area on narrow screen sizes. The second fix keeps the `TaskArea` and `ImageToolbar` heights equal. 

On issue #561, the error occurs due to the `pad: small` designation on the Grommet `Box` changing heights when the 768 responsive breakpoint is hit. The SVGs oddly try to compensate by growing in size, thus stretching the SVG icon, and toolbar, to the size shown in that issue.

# Review Checklist

## General

- [x] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [x] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [x] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/MetaTools/MetaTools.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Box } from 'grommet'
 import { inject, observer } from 'mobx-react'
+import styled from 'styled-components'
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/MetaTools/MetaTools.js]: I don't think you need this

File: packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js [hunk 2]
```diff
@@ -3,12 +3,17 @@ import { Box, Paragraph } from 'grommet'
...
 import React from 'react'
-import { ThemeProvider } from 'styled-components'
+import styled, { ThemeProvider } from 'styled-components'
 
 import getTaskComponent from './helpers/getTaskComponent'
 import TaskHelp from './components/TaskHelp'
 import { default as TaskNavButtons } from './components/TaskNavButtons'
 
+const StyledBox = styled(Box)`
+  max-height: 600px;
```
Comment (rogerhutchings) [inline packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js]: Isn't this just going to cause content clipping if we have a task with lots of choices?

Comment (wgranger) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js]: Ah yes. I meant to add an overflow here.

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js]: I realized this was a bad idea in my comment: https://github.com/zooniverse/front-end-monorepo/issues/561#issuecomment-474437971

Volunteers might miss tasks this way. It's should be a min-height instead that allows the container to grow.

Comment (wgranger) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js]: Gotcha. I was thinking we were using a max-height and encouraging users to stick to max task numbers with overflow as a safety net. I've pushed a new commit removing max height. The container should grow as needed.

Comment (srallen) [reply packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js]: We can maybe do that once the workflow editor page is redesigned, but that's a ways off still.

**Comment by srallen** [comment]
I've rebased this to fix the conflicts

**Comment by srallen** [comment]
@rogerhutchings I think I fixed those things. We might want to consider refactoring the default layout to use Grommet's ResponsiveContext so that it uses the same breakpoints as other components we've started to use it with.

**Comment by rogerhutchings** [comment]
Cool, will do another check. We should also add those breakpoints into the theme at some point

**Comment by srallen** [comment]
Just rebased with the latest master.

**Comment by srallen** [comment]
I'm making sure a few things in #695 get into this PR.

**Comment by srallen** [comment]
You may notice there's an extra gap in the metatools area between the subject metadata button and the favorites button. This is because we're hiding the modal there. I'd like to refactor that to get rid of this and have a single container fragment component for the button and modal like I did with the field guide. 

@rogerhutchings could I get another review on this? There's a few additional fixes for #695 here.

**Pull request merged**